### DISCUSSION TITLE
Plugin traductor del blog

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#37] [Announcements]
# Plugin traductor del blog

Agrega un traductor personalizado a tu Blog potenciado por Google Translate.

https://www.zkreations.com/2024/02/gadget-traductor-del-blog.html

**Comment by MioYonikiru** [comment]
Hola! una pregunta, hay alguna forma de hacer que el font family se mantenga tal cual durante la traducción? ya que me cambia el formato y termina viéndose mal. En algunas partes le deje el notranslate pero no se podría aplicar a todo.

<img src="https://i.imgur.com/c8OyJdV.png">

**Reply by danieIabel** [reply]
Hola amigo disculpa, recien veo tu pregunta, una sincera disculpa. Este plugin usa enteramente la api de Google Translate, y esta misma al momento de hacer traducciones envuelve el texto traducido en una etiqueta `<font>` probablemente en tu diseño esa etiqueta no tenga estilos o se esté aplicando una fuente por defecto. 

Otra cosa que puede influir es que la fuente que uses no disponga de los caracteres especiales para otros idiomas. Se que esto ultimo no es tu caso pero quería dejar la informacion por si alguien mas lo lee.
